app/renderer/assets/locales/en-US/global.json

```diff
@@ -23,7 +23,7 @@
   "error_not_valid_stake_amount": "Not a valid Stake Amount (must have be a positive amount with a maximum of 4 decimals of precision eg. 12.1212).",
   "error_ram_has_to_be_over_minimum_amount": "A minimum of 3 bytes of RAM is required per transaction.",
   "error_ram_not_valid_amount": "Invalid RAM amount. A valid amount should be a whole number with no decimals.",
-  "error_ram_under_minimum_for_new_account": "A minimum of 4000 Bytes of RAM is required to create a new account.",
+  "error_ram_under_minimum_of_2600_for_new_account": "A minimum of 2600 Bytes of RAM is required to create a new account.",
```

Comment:
Hrm - do you have any concerns about crowdin not liking the removal of the string?  I'm on the fence, but it might be best if we left the old string and made the new one for now. 

Should probably also find a utility to find "unused strings" at some point too :)

Reply:
Hmm I assumed that it would be fine with it, but you got me nervous about it now 😄We should probably keep it in to be safe.